src/qualifications/presenters/qualification.presenter.ts

```diff
@@ -9,19 +9,24 @@ export default class QualificationPresenter {
     private readonly i18nService: I18nService,
   ) {}
 
-  readonly level = this.qualification.level;
+  readonly level = this.qualification && this.qualification.level;
```

Comment:
Would safe navigation be possible here? i.e:

```suggestion
  readonly level = this?.qualification.?level;
```

Reply:
👍